### PR TITLE
Update tele_sight description

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: Content "Describe telescopic sight as post apocalyptic"```

#### Purpose of change
Since it can be crafted by a survivor it should be described as such to prevent it adding up in itemgroups that contain pre-cataclysm items.

#### Describe the solution
Describe the telescopic sight as post cataclysm

#### Describe alternatives you've considered
#34755

#### Testing
image

#### Additional context
Follow up on #34755

### CODE REVIEW

File: data/json/items/gunmod/sights.json [hunk 1]
```diff
@@ -185,7 +185,7 @@
     "id": "tele_sight",
     "type": "GUNMOD",
     "name": "telescopic sight",
-    "description": "A simple telescopic sight, essentially a small telescope with crosshairs.  Increases weight but improves accuracy.",
+    "description": "A simple hand-crafted telescopic sight, essentially a small telescope with crosshairs. Not as good as the ones made before the cataclysm.  Increases weight but improves accuracy.",
```
Comment (AMurkin) [inline data/json/items/gunmod/sights.json]:
```suggestion
    "description": "A simple hand-crafted telescopic sight, essentially a small telescope with crosshairs.  Not as good as the ones made before the cataclysm.  Increases weight but improves accuracy.",
```

Comment (tenmillimaster) [reply data/json/items/gunmod/sights.json]: Is it just me or is calling it 'the cataclysm' a little 4th wall breaking? I find it pretty jarring, I'd go for "before the dead rose" or something.

Comment (AMurkin) [reply data/json/items/gunmod/sights.json]: We have plenty mentions of the cataclysm.

Comment (snipercup) [reply data/json/items/gunmod/sights.json]: Why would there be 2 spaces after a period?

Comment (anothersimulacrum) [reply data/json/items/gunmod/sights.json]: That's convention in the game. I believe it's due to monospaced font style rules, which require two spaces after a period.

**Comment by snipercup** [comment]
Please merge.

**Pull request merged**